hawc/apps/mgmt/templates/mgmt/user_assessment_assignments.html

```diff
@@ -1,10 +1,78 @@
-{% extends 'assessment-rooted.html' %}
-
-{% block content %}
+{% extends 'assessment-rooted.html' %} {% block content %}
+<div>
     <h2>My assigned tasks</h2>
-    <div id='main'></div>
-{% endblock %}
 
-{% block extrajs %}
-    {{ config|json_script:"config" }}
-{% endblock extrajs %}
+    {% if show_completed %}
+    <a class="btn btn-light" style="margin-bottom: 15px" role="button" href=".?completed=off">Hide completed and
+        abandoned tasks</a>
+    {% else %}
+    <a class="btn btn-light" style="margin-bottom: 15px" role="button" href=".?completed=on">Show completed and
+        abandoned tasks</a>
+    {% endif %}
+
+    {% if object_list %}
+    <div>
+        <table class="table table-sm table-striped">
+            <colgroup>
+                <col width="30%" />
+                <col width="70%" />
+            </colgroup>
+            <thead>
+                <tr>
+                    <th>Study</th>
+                    <th>Task</th>
+                </tr>
+            </thead>
+            <tbody>
+                {% for task in object_list %}
+                <tr>
+                    <td>
+                        <a href="{{ task.study.get_absolute_url }}">{{ task.study.short_citation }}</a>
+                    </td>
+                    {% include "mgmt/fragments/task_cell.html" with object=task %}
+                </tr>
+                {% endfor %}
+            </tbody>
+        </table>
+    </div>
+    {% else %}
+    <div>
+        <i>There are no tasks available</i>
+    </div>
+    {% endif %}
+
+    <!-- risk of bias section  -->
+    <div>
+        <h4>Pending risk of bias/study evaluation reviews</h4>
+        {% if rob_task_list %}
+        <table class="table table-sm table-striped">
+            <colgroup>
+                <col width="33%" />
+                <col width="66%" />
+            </colgroup>
+            <thead>
+                <tr>
+                    <th>Study</th>
+                    <th>Action</th>
+                </tr>
+            </thead>
+            <tbody>
+                {% for task in rob_task_list %}
+                <tr>
+                    <td>
+                        <a href="{{ task.study.get_absolute_url }}">{{ task.study.short_citation }}</a>
+                    </td>
+                    <td>
+                        <a class="btn btn-light" href="{{task.get_edit_url}}"><i class="fa fa-edit"></i> Edit review</a>
+                        <!-- get_edit_url does not work with current query  -->
```

Comment:
remove comment? or is this an issue that we still need to fix?

Reply:
it looks like in views, the query set gets a task model instance but we need to query the risk of bias model in order to call its get_edit_url function. this is what `get_review_tasks2` achieves. should I replace that with the current query?